internal/pwru/utils.go

```diff
@@ -15,15 +17,45 @@ import (
 
 type Funcs map[string]int
 
+// getAvailableFilterFunctions return list of functions to which it is possible
+// to attach kprobes.
+func getAvailableFilterFunctions() (map[string]struct{}, error) {
+	availableFuncs := make(map[string]struct{})
+	path := "/sys/kernel/debug/tracing/available_filter_functions"
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open %s: %v", path, err)
```

Comment:
Nit: I think we can return the plain `err` here, `os.Open` will return an `os.PathError` which already contains the path.

```suggestion
		return nil, err
```